src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.tag;
+
+/**
+ * Association Class for Tag
+ */
+public class Tagging {
+    
+    public Tag tag;
```

Comment:
the association that Tagging class should maintain is between the person and the tag, right?
in your implementation here, how do you capture the person details?